Makefile

```diff
@@ -81,7 +84,7 @@ __check_defined = \
 
 .PHONY: $(GOPATH)/bin/minishift$(IS_EXE)
 $(GOPATH)/bin/minishift$(IS_EXE): $(ADDON_ASSET_FILE) ## Builds the binary into $GOPATH/bin
-	go install -tags "$(BUILD_TAGS)" -pkgdir=$(ADDON_BINDATA_DIR) -ldflags="$(VERSION_VARIABLES)" ./cmd/minishift
+	go install -tags "$(BUILD_TAGS_SYSTEMTRAY)" -pkgdir=$(ADDON_BINDATA_DIR) -ldflags="$(VERSION_VARIABLES)" ./cmd/minishift
```

Comment:
@anjannath What is this mean, Shouldn't this install systemtray every time `make` is run. But thats not happening. So what am I missing?

Reply:
@LalatenduMohanty Yes, now when you have mentioned it, this will be modified to use `go install`